src/praisonai-agents/praisonaiagents/llm/model_capabilities.py

```
@@ -101,14 +161,14 @@ def supports_parallel_function_calling(model_name: str) -> bool:
     try:
         litellm = _get_litellm()
         if litellm is None:
-            return False
+            return _fallback_supports_function_calling(model_name)
         # Use LiteLLM's built-in check - most accurate and up-to-date
         if hasattr(litellm, 'supports_parallel_function_calling'):
             return litellm.supports_parallel_function_calling(model=model_name)
     except Exception:
         pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)